filebeat/module/system/audit/ingest/pipeline.json

```diff
@@ -0,0 +1,117 @@
+{
+    "description": "Pipeline for parsing Linux audit logs",
+    "processors": [
+        {
+            "grok": {
+                "field": "message",
+                "pattern_definitions": {
+                    "AUDIT_TYPE": "^type=%{NOTSPACE:system.audit.type}",
+                    "AUDIT_PREFIX": "%{AUDIT_TYPE} msg=audit\\(%{NUMBER:system.audit.epoch}:%{NUMBER:system.audit.counter}\\):(%{DATA})?",
+                    "AUDIT_KEY_VALUES": "%{WORD}=%{GREEDYDATA}"
+                },
+                "patterns": [
+                    "%{AUDIT_PREFIX} %{AUDIT_KEY_VALUES:system.audit.kv} old auid=%{NUMBER:system.audit.old_auid} new auid=%{NUMBER:system.audit.new_auid} old ses=%{NUMBER:system.audit.old_ses} new ses=%{NUMBER:system.audit.new_ses}",
+                    "%{AUDIT_PREFIX} %{AUDIT_KEY_VALUES:system.audit.kv} msg=['\"](%{DATA:system.audit.msg}\\s+)?%{AUDIT_KEY_VALUES:system.audit.sub_kv}['\"]",
+                    "%{AUDIT_PREFIX} %{AUDIT_KEY_VALUES:system.audit.kv}",
+                    "%{AUDIT_PREFIX}",
+                    "%{AUDIT_TYPE} %{AUDIT_KEY_VALUES:system.audit.kv}"
+                ]
+            }
+        },
+        {
+            "kv": {
+                "field": "system.audit.kv",
+                "field_split": "\\s+",
+                "value_split": "=",
+                "target_field": "system.audit"
+            }
+        },
+        {
+            "kv": {
+                "field": "system.audit.sub_kv",
+                "field_split": "\\s+",
+                "value_split": "=",
+                "target_field": "system.audit",
+                "ignore_missing": true
+            }
+        },
+        {
+            "remove": {
+                "field": "system.audit.kv",
+                "ignore_failure": true
+            }
+        },
+        {
+            "remove": {
+                "field": "system.audit.sub_kv",
+                "ignore_failure": true
+            }
+        },
```

Comment:
One reason I like having the message field is because it gets analyzed and make searching simpler when I don't know exactly what I'm looking for (helps with `_all` being deprecated). WDYT?